docs/service/webapp.rst

```diff
@@ -58,40 +58,18 @@ Run this::
 You can do this as many times as you like.
 
 
-Setting up the webapp for Google Sign-In
-----------------------------------------
+Setting up the webapp for OpenID Connect Login
+----------------------------------------------
 
-In order to authenticate in the webapp using the "Google signin" button, you
-need to get a set of oauth credentials.
+In order to authenticate in the webapp using the "Login" button, you
+need to add the following entry to your ``/etc/hosts`` (or equivalent) file:
 
-1. Go to https://console.developers.google.com/apis/credentials
+```
+127.0.0.1 oidcprovider
+```
```

Comment:
This is the Markdown syntax for a literal block, but this file (and the rest of the docs) are in [reStructuredText](http://docutils.sourceforge.net/docs/user/rst/quickstart.html) format. We'll want to use the syntax for a preformatted block for rst:

```rst
In order to authenticate in the webapp using the "Login" button, you
need to add the following entry to your ``/etc/hosts`` (or equivalent) file::
  
   127.0.0.1 oidcprovider
```

Note the double colon at the end of the preceding paragraph. That plus the indent is the syntax in rst for a preformatted block of text.